src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -31,6 +32,24 @@ class AutowirePass extends AbstractRecursivePass
     private $ambiguousServiceTypes = array();
     private $autowired = array();
     private $lastFailure;
+    private $throwOnAutowiringException;
+    private $autowiringExceptions = array();
+
+    /**
+     * @param bool $throwOnAutowireException If false, retrieved errors via getAutowiringExceptions
+     */
+    public function __construct($throwOnAutowireException = true)
```

Comment:
Exactly - for BC if you're using the class directly. Seems like an unlikely thing, but Nicolas mentioned it... and technically, we *do* need this for BC.